lua/starfall/libs_sh/entities.lua

```diff
@@ -1286,4 +1286,20 @@ function ents_methods:getHitBoxHitGroup(hitbox, hitboxset)
 	return getent(self):GetHitBoxHitGroup(hitbox, hitboxset)
 end
 
-end
\ No newline at end of file
+--- Returns a table of brushes surfaces for brush model entities.
+-- @shared
+-- @class function
+-- @return Table of SurfaceInfos if the entity has a brush model, or no value otherwise.
+function ents_methods:getBrushSurfaces()
```

Comment:
- `out` should be localized
- `swrap` is `nil`
- The function should return `no value` if the entity has no brush surfaces, as the description says
- Change all `for` loop variables to be in camelCase

Reply:
Can you make it return `no value` when the entity has no brush surfaces, as the original function does, so if you need a table you can just do `ent:getBrushSurfaces() or {}`?

Reply:
Should now return no value